dev_guide/managing_images.adoc

```diff
@@ -586,6 +586,235 @@ instead of listing all image streams.
 ====
 endif::openshift-origin,openshift-enterprise,openshift-dedicated[]
 
+[[auth-enabled-registry]]
+== Authentication Enabled Red Hat Registry
+
+//Concept
+All container images available through the Red Hat Container Catalog are hosted
+on an image registry, `registry.access.redhat.com`. The Red Hat Container Catalog
+is moving from `registry.access.redhat.com` to `registry.redhat.io`.
+
+The new registry, `registry.redhat.io`, requires authentication for access to
+images and hosted content on {product-title}. Following the move to the new
+registry, the existing registry will be available for a period of time.
+
+The new registry uses standard OAuth mechanisms for authentication,
+with the following methods:
+
+* *Web username and password.* This is the standard set of credentials you use
+to log in to resources such as `access.redhat.com`.
+* *Authentication token.* These tokens, generated by administrators, are non-user
+accounts that give systems the ability to authenticate against
+the container registry.
+
+You can use `docker login` with your credentials, username and password or authentication
+token, to access content on the new registry.
+
+All image streams are updated to point to the new registry. Because the
+new registry requires authentication for access, there is a new secret in the
+{product-title} namespace called `imagestreamsecret`. When you run your build,
+your `imagestreamsecret` gets populated with your `kubernetes.io/dockerconfigjson`
+credentials.
+
+There are two places your credentials must exist:
+
+* *{product-title} namespace*. Your credentials must exist in the {product-title}
+namespace so that the image streams in the {product-title} namespace can import.
+* *Your host*. Your credentials must exist on your host because Kubernetes will
+use the credentials from your host when it goes to pull images.
```

Comment:
both of these things should be setup for you by the ansible installer.  it's worth explaining they need to be there, but it should also be clear that the ansible install will set it up for you (assuming you populate all the correct ansible vars.  Since those vars are in flux right now you're going to need feedback from @sdodson and @michaelgugino to make sure the exact var names are correct)